docs/index.md

```diff
@@ -23,7 +23,15 @@
 
 > A rising tide lifts all boats. -- United States President, John F. Kennedy (borrowed from the New England Council)
 
-Inspired by the proverb _“A rising tide lifts all boats”_, when we lower the barrier of entry to writing and choosing quality code for enough people, it will lift the quality of code across the whole WordPress ecosystem. Tide’s vision is to make it easy to improve the quality of code throughout the WordPress ecosystem and help WordPress site owners make better choices about plugins and themes.
+Inspired by the proverb _“A rising tide lifts all boats”_, we want to lift the quality of code across the whole WordPress ecosystem. Tide’s vision is to run automated quality testing for all WordPress plugins and themes and make those test results visible for both the authors and to end users of those plugins and themes. This will help improve the quality of code throughout the WordPress ecosystem and help WordPress site owners make better choices about plugins and themes.
+
+## Status summer 2019
+
+The project is an official part of the WordPress project, but still in the early stages. Currently there is in place a system that scans all WordPress plugins and themes and an API that provides the scan results as JSON output.
+
+Next steps is to start using the JSON output in the WordPress plugin repository to show PHP compatibility information. This will help drive the adoption of newest PHP versions as both plugin authors and WordPress site owners instaling plugins will be more aware of PHP compatibility of each plugin.
```

Comment:
```suggestion
The next step is to start using the JSON output in the WordPress plugin repository to show PHP compatibility information. This will help drive the adoption of newest PHP versions as both plugin authors and WordPress site owners installing plugins will be more aware of the PHP compatibility of each plugin.
```